### PR TITLE
TURTLES-645: Load maas_rally plugin configuration via class

### PR DESCRIPTION
**Pull request by cfarquhar**
This change adds a class to manage plugin configuration in preparation for more
steps in the plugin relying on configuration read from a file.

### CODE REVIEW

File: playbooks/files/rax-maas/plugins/rally_performance.py [hunk 1]
```diff
@@ -248,4 +267,5 @@ def main():
     parser = make_parser()
     args = parser.parse_args()
     with maas_common.print_output(print_telegraf=args.telegraf_output):
+        plugin_config = PluginConfig(PLUGIN_CONF)
```
Comment (tonytan4ever) [inline playbooks/files/rax-maas/plugins/rally_performance.py]: It seems odd as plugin_config is used as a kind of global variable, I think it's better to pass it into the main function main(plugin_config), and in turn passed it in send_metrics_to_influxdb(plugin_config), what do you think?

Comment (cfarquhar) [reply playbooks/files/rax-maas/plugins/rally_performance.py]: Thanks for your feedback.  I've updated the PR as you suggested.

**Comment by cfarquhar** [comment]
recheck_cit_trusty_newton_deploy

**Comment by cfarquhar** [comment]
TURTLES-645

**Comment by tonytan4ever** [comment]
LGTM, merging

**Pull request merged**